harness/determined/experimental/_native.py

```diff
@@ -235,6 +235,9 @@ def create(
             An optional string to use as the Determined master URL when
             ``local=False``. If not specified, will be inferred from the
             environment variable ``DET_MASTER``.
+
+    Returns:
+        int: the created experiment ID.
```

Comment:
Can you share a code snippet of how you are intending to use this in your scripts?

This is a change to a deprecated API.  It's not out of the question but it would help to see what you're trying to use it for.

Reply:
Ahh dang, I didn't know this was deprecated! What it's replaced by?

Use case is to trigger, wait on, and get results of an experiment using the SDK as part of a) some automated workflows and b) integration tests.